server/publicapi/items/service.py

```diff
@@ -26,6 +28,60 @@ class ItemsService(BaseService):
     Serves mainly as a proxy to the data layer.
     """
 
+    def find_one(self, req, **lookup):
+        """Retrieve a specific item.
+
+        :param req: object representing the HTTP request
+        :type req: `eve.utils.ParsedRequest`
+        :param dict lookup: requested item lookup, contains its ID
+
+        :return: requested item (if found)
+        :rtype: dict or None
+        """
+        request_params = req.args or {}
+        params = list(request_params.keys())
+
+        if len(params) > 0:
```

Comment:
I think it would be better to split parameter checking in a different method, find_one and get should contain only business logic. Also, why not name the query parameter "query"?